docs/source/whatsnew/1.0.2.txt

```diff
@@ -0,0 +1,23 @@
+Release 1.0.2
+-------------
+
+:Release: 1.0.2
+:Date: TBD
+
+Enhancements
```

Comment:
eh, I think it is more of a feature because you will need to do work to take advantage of it.